### DISCUSSION TITLE
这个项目越来越火了，会不会被制裁啊，像是 pandownload 一样

### DISCUSSION THREAD

**Discussion by wgx-dh** [AlistGo/alist#1959] [General]
B 站 40 多万播放量的视频教大家用 Alist 😂

**Comment by AspirantH** [comment]
1、开源
2、无利益关联
以上

**Comment by xmxu2022** [comment]
这个就是一个正常的第三方软件呗，跟你用浏览器去看网盘是一样的逻辑和原理，不存在灰色地带

**Comment by ENuiOWyTv55o** [comment]
pandownload 是真的影响到了百度的收益，毕竟人家是要靠限速迫使人家开会员的。alist又不解决限速问题，你不开会员用这玩意也是毫无体验的。

**Comment by xinxinenjoy** [comment]
这个顶多属于方便了用户，而且对其他任何人都没有影响
